code/go/0chain.net/chaincore/chain/state.go

```diff
@@ -205,6 +205,11 @@ func (c *Chain) updateState(ctx context.Context, b *block.Block, bState util.Mer
 	// check if the block's ClientState has root value
 	_, err = bState.GetNodeDB().GetNode(bState.GetRoot())
 	if err != nil {
+		logging.Logger.Error("update_state_failed",
```

Comment:
The caller will log the error, let's not duplicate the logging

Reply:
removed